pkgs/by-name/an/annotator/package.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, meson
+, cmake
+, ninja
+, vala
+, wrapGAppsHook
+, desktop-file-utils
+, libgee
+, pantheon
+, libxml2
+, libhandy
+}:
+
+stdenv.mkDerivation rec {
+  pname = "annotator";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "phase1geo";
+    repo = pname;
+    rev = version;
+    hash = "sha256-VHvznkGvrE8o9qq+ijrIStSavq46dS8BqclWEWZ8mG8=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    meson
+    cmake
```

Comment:
Do we really need cmake, ninja, pkg-config and meson? Usually three of the four are enough and the warnings about other missing can be ignored.

Reply:
Thanks for noticing. Cmake can be safely ignored.